### ISSUE TITLE
'etc/unknown' timezone should fall back on UTC

### ISSUE THREAD

**Issue by guillaumelachaud** [spencermountain/spacetime#367]
On some systems with incorrect (missing) timezone setup, `Intl.DateTimeFormat().resolvedOptions().timeZone` will return `etc/unknown`, which in turn will throw an exception when trying to lookup the timezone:

```
Spacetime: Cannot find timezone named: 'etc/unknown'. Please enter an IANA timezone id.
```

It feels like `etc/unknown` should be treated as `undefined` in `safeIntl()`, allowing `guezzTz()` to gracefully fall back on UTC. 

Thoughts?

**Comment by spencermountain** [comment]
whoa! I did not know that. That's helpful to know.
Yeah - maybe we should map it back to UTC?
Thanks Guillaume.

**Comment by guillaumelachaud** [comment]
I think it would make sense to map it back to UTC, yes.

I see that you added commit [d51a5fe](https://github.com/spencermountain/spacetime/commit/d51a5fe6b972ad719c8b0f15a07c834e9cc6b3ed) on a separate branch, which name hints at a major rewrite :-). Any plans to add a fix for this in master as well ?

Thanks!

**Comment by spencermountain** [comment]
sure. released as 7.4.1 
cheers
